lib/components/src/tooltip/WithTooltip.js

```diff
@@ -32,19 +32,13 @@ const WithTooltip = ({
 }) => {
   const Container = svg ? TargetSvgContainer : TargetContainer;
 
-  if (tooltipShown !== undefined) {
-    Object.assign(props, { tooltipShown });
-  }
-  if (onVisibilityChange !== undefined) {
-    Object.assign(props, { onVisibilityChange });
-  }
-
   return (
     <TooltipTrigger
       placement={placement}
       trigger={trigger}
       modifiers={modifiers}
-      {...props}
```

Comment:
We shouldn't pass `props` both too the `TooltipTrigger` *and* the `Container` (below). It should be one of the other. @domyen which one makes sense to you?

Reply:
The only thing I use `...props` for is styled-components. Alternatively, if we're not passing props here it might make more sense to explicitly use `className`

Reply:
@domyen my point was previously this component was doing that in two points (the `TooltipTrigger` *and* the `Container`). It seems wrong (and it caused problems) to do that